src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -608,12 +656,45 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 		});
 	}
 
+	private showContextViewHelper(menu: CodeActionMenu, params: ICodeActionMenuParameters, menuActions: IAction[]) {
+		params.menuObj._contextViewService.showContextView({
```

Comment:
Do you need to pass in the `_contextViewService` here or can you use `this.contextViewService`?